my-app/src/chatWindow/services/chatMapperService.js

```diff
@@ -0,0 +1,23 @@
+class ChatMapperService {
+	mapMyMessage = (textMyMessage, AUTHOR) => {
+		return [
+			{
+				TEXT_MESSAGE: textMyMessage,
+				DATE_MESSAGE: Date.now(),
+				AUTHOR: AUTHOR[0],
+			},
+		];
+	};
+
+	mapNotMyMessage = (json, AUTHOR) => {
```

Comment:
if I understand correctly, it should be as mapToUI (as we send ui model to the ui only)